examples/pylab_examples/print_stdout.py

```diff
@@ -1,13 +1,9 @@
-#!/usr/bin/env python
-# -*- noplot -*-
-# print png to standard out
```

Comment:
Can you leave in the last to lines. They explain what the example is supposed to do